src/frontend/src/handler/create_sql_function.rs

```diff
@@ -176,12 +176,27 @@ pub async fn handle_create_sql_function(
             .update_context(create_mock_udf_context(arg_types.clone()));
 
         if let Ok(expr) = UdfContext::extract_udf_expression(ast) {
-            if let Err(e) = binder.bind_expr(expr) {
+            let bind_result = binder.bind_expr(expr);
+
+            if let Err(e) = &bind_result {
                 return Err(ErrorCode::InvalidInputSyntax(format!(
                     "failed to conduct semantic check, please see if you are calling non-existence functions: {}",
                     e.as_report()
                 ))
                 .into());
+            } else {
+                // We can safely unwrap here
+                let e = bind_result.unwrap();
```

Comment:
Try `match` to avoid `unwrap`.
```rust
let bound_stmt = match binder.bind_expr(expr) {
    Ok(res) => res,
    Err(e) => return Err(...),
};
```